voxel51/platform/api.py

```diff
@@ -187,20 +190,24 @@ def upload_job_output_as_data(self, job_id, path):
         _validate_response(res)
         return _parse_json_response(res)["data"]["data_id"]
 
-    def _get_job_url(self, job_id, url_type):
+    def _get_job_url(self, task_config, url_type):
         '''Retrieves a signed URL to post job information or output.
 
         Args:
-            job_id (str): the job ID
+            task_config (voxel51.platform.task.TaskConfig): the task config
             url_type (str): one of "status", "log", or "output"
 
         Returns:
             a RemotePathConfig object
         '''
-        endpoint = self.base_url + "/jobs/" + job_id + "/url/" + url_type
+        endpoint = \
+            self.base_url + "/jobs/" + task_config.job_id + "/url/" + url_type
         res = self._requests.get(endpoint, headers=self._header)
-        _validate_response(res)
-        return voxu.RemotePathConfig(_parse_json_response(res))
+        try:
+            _validate_response(res)
+            return voxu.RemotePathConfig(_parse_json_response(res))
+        except:
```

Comment:
Same here (APIError is defined in this file, right below, so no need for extra imports)